README.md

```diff
@@ -496,151 +689,94 @@ Babel 7 supports the decorators proposal presented to TC39 in the November 2018
 
 #### Syntax changes
 
-On the side of using decorators, this proposal makes several changes compared to previous decorators proposals:
-- When importing a decorator from a module, include `@` as part of the name of the decorator; previous proposals excluded the `@` during an import.
+On the side of using decorators, this proposal makes a couple changes compared to previous decorators proposals:
 - Forms like `@foo.bar` or `@(foo)` are no longer permitted, as decorators are not JavaScript expressions.
```

Comment:
I think this description of the syntax needs to be updated to follow the new version. (We already have `Syntax` section above. So I think we can delete this `Syntax changes` section in FAQ.

Reply:
Thanks, you're right; deleted this section.